doc/maintaining/background-tasks.rst

```diff
@@ -147,7 +147,7 @@ Afterwards the worker waits again for the next job to be enqueued.
 
 Workers can be started using the :ref:`paster jobs worker` command::
 
-    paster --plugin=ckan jobs worker --config=/etc/ckan/default/development.ini
+    ckan -c /etc/ckan/default/ckan.ini --plugin=ckan jobs worker 
```

Comment:
There is no need in `--plugin` option. Moreover, it's not supported, so here we need `ckan -c /etc/ckan/default/ckan.ini jobs worker`
Same for lines 204 and 273